packages/tokens/data/icons.json

```diff
@@ -0,0 +1,763 @@
+{
+    "2:61718": "<svg width=\"16\" height=\"16\" viewBox=\"0 0 16 16\" fill=\"none\" xmlns=\"http://www.w3.org/2000/svg\">\n<circle cx=\"6.5\" cy=\"6.5\" r=\"5\" stroke=\"#282829\"/>\n<path d=\"M14.5 14.5044L10.2573 10.2618\" stroke=\"#282829\" stroke-linecap=\"round\" stroke-linejoin=\"round\"/>\n<path d=\"M4.5 6.5H8.5\" stroke=\"#282829\" stroke-linecap=\"round\"/>\n</svg>\n",
+    "2:61724": "<svg width=\"24\" height=\"24\" viewBox=\"0 0 24 24\" fill=\"none\" xmlns=\"http://www.w3.org/2000/svg\">\n<circle cx=\"10\" cy=\"10\" r=\"7.5\" stroke=\"#282829\"/>\n<path d=\"M21.5 21L15.8432 15.3431\" stroke=\"#282829\" stroke-linecap=\"round\" stroke-linejoin=\"round\"/>\n<path d=\"M7 10H13\" stroke=\"#282829\" stroke-linecap=\"round\"/>\n</svg>\n",
```

Comment:
Is it normal that in this file the attributes are not cleaned?

Reply:
Yes, this is the raw data we receive from figma API

Reply:
Ooops, thanks for the precision ;)